web/rest/app/config/api/raw/provider.yml

```diff
@@ -3,138 +3,401 @@
   attributes:
-    access_control: '"ROLE_ADMIN" in roles'
+    access_control: '"ROLE_COMPANY_ADMIN" in roles'
+    read_access_control:
+      inherited:
+        conditionalRoute: 'Ivoz\Provider\Domain\Model\ConditionalRoute\ConditionalRoute'
 Ivoz\Provider\Domain\Model\ConditionalRoutesConditionsRelMatchlist\ConditionalRoutesConditionsRelMatchlist:
+  collectionOperations:
+    get:
+      operation_normalization_context: 'detailedCollection'
+      normalization_context:
+        groups: ['detailedCollection']
   attributes:
-    access_control: '"ROLE_ADMIN" in roles'
+    access_control: '"ROLE_COMPANY_ADMIN" in roles'
+    read_access_control:
+      inherited:
+        matchlist: 'Ivoz\Provider\Domain\Model\MatchList\MatchList'
 Ivoz\Provider\Domain\Model\ConditionalRoutesConditionsRelSchedule\ConditionalRoutesConditionsRelSchedule:
+  collectionOperations:
+    get:
+      operation_normalization_context: 'detailedCollection'
+      normalization_context:
+        groups: ['detailedCollection']
   attributes:
-    access_control: '"ROLE_ADMIN" in roles'
+    access_control: '"ROLE_COMPANY_ADMIN" in roles'
+    read_access_control:
+      inherited:
+        condition: 'Ivoz\Provider\Domain\Model\ConditionalRoutesCondition\ConditionalRoutesCondition'
 Ivoz\Provider\Domain\Model\ConditionalRoutesConditionsRelCalendar\ConditionalRoutesConditionsRelCalendar:
+  collectionOperations:
+    get:
+      operation_normalization_context: 'detailedCollection'
+      normalization_context:
+        groups: ['detailedCollection']
   attributes:
-    access_control: '"ROLE_ADMIN" in roles'
+    access_control: '"ROLE_COMPANY_ADMIN" in roles'
+    read_access_control:
+      inherited:
+        calendar: 'Ivoz\Provider\Domain\Model\Calendar\Calendar'
 Ivoz\Provider\Domain\Model\ConferenceRoom\ConferenceRoom:
   attributes:
-    access_control: '"ROLE_ADMIN" in roles'
+    access_control: '"ROLE_COMPANY_ADMIN" in roles'
+    read_access_control:
+      ROLE_COMPANY_ADMIN:
+        company:
+          eq: "user.getCompany().getId()"
+      ROLE_BRAND_ADMIN:
+        company:
+          in: "companyRepository.getSupervisedCompanyIdsByAdmin(user)"
+Ivoz\Provider\Domain\Model\Country\Name:
+Ivoz\Provider\Domain\Model\Country\Zone:
 Ivoz\Provider\Domain\Model\Country\Country:
+  itemOperations:
+    put: ~
+    delete: ~
+  collectionOperations:
+    post: ~
   attributes:
-    access_control: '"ROLE_SUPER_ADMIN" in roles'
+    access_control: '"ROLE_USER" in roles'
 Ivoz\Provider\Domain\Model\Ddi\Ddi:
   attributes:
-    access_control: '"ROLE_ADMIN" in roles'
+    access_control: '"ROLE_COMPANY_ADMIN" in roles'
+    read_access_control:
+      ROLE_COMPANY_ADMIN:
+        company:
+          eq: "user.getCompany().getId()"
+      ROLE_BRAND_ADMIN:
+        company:
+          in: "companyRepository.getSupervisedCompanyIdsByAdmin(user)"
 Ivoz\Provider\Domain\Model\Domain\Domain:
   attributes:
     access_control: '"ROLE_SUPER_ADMIN" in roles'
```

Comment:
Like Countries, this is a readonly table that is populated during Brand/Company lifecycles.

Conditions to access its contents are not easy define because it has no company nor brand fields. Multiple entities (Brands, Companies, Terminals, RetailAccounts) has a domain field and that's the only domain they can read.